docs/example/manifests/apps/bookstore-v2.yaml

```diff
@@ -44,6 +44,17 @@ spec:
       nodeSelector:
         kubernetes.io/arch: amd64
         kubernetes.io/os: linux
+      initContainers:
```

Comment:
is this init container trying to synchronize the deployment of the pods? i.e. bookstore pod does not start until the database pod is up

Reply:
Yes. This init container polls the mysql service until it's up.

Reply:
I am not excited about this because it breaks windows demo. Is there a way to make the scheduling work from a bash script or to have the bookstore be resilient to database connection failures?

Reply:
That's possible. @allenlsy would you mind dropping the init-container and have the bookstore poll the database within the application itself, and retry the connections in a poll loop. The loop will eventually break when there is a policy that permits the bookstore to connect to the DB.

Reply:
Yes, I have implemented pre-condition checking in the bookstore golang code. This init container can be removed.